Gopkg.toml

```diff
@@ -34,285 +34,489 @@ required = [
 
+  # main-usage = "plugins/cilium-docker"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/evalphobia/logrus_fluent"
   revision = "v0.4.0"
 
+  # main-usage = "pkg/logging"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/fatih/color"
   revision = "533cd7fd8a85905f67a1753afb4deddc85ea174f"
 
+  # main-usage = "pkg/colors"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/gogo/protobuf"
   revision = "v0.4-3-gc0656ed"
 
+  # main-usage = "consider to remove this dependency"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/golang/protobuf"
   revision = "5831880292e721c76b58a16ecc60adc27d8e6355"
 
+  # main-usage = "pkg/envoy and proxylib"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/google/gopacket"
   revision = "632566d585d0456397d2058bd9cb800c48e3230b"
 
+  # main-usage = "pkg/monitor"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/google/gops"
   revision = "25312cafb9a191a6d377c480a4666beec3105e4a"
 
+  # main-usage = "all main packages"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/errors"
   revision = "fc3f73a224499b047eda7191e5d22e1e9631e86f"
 
+  # main-usage = "api/v1"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/loads"
   revision = "bf98340e04734f11563fd3008e87be4c1d4ffc08"
 
+  # main-usage = "swagger spec"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/runtime"
   revision = "bf2ff8f7150788b1c7256abb0805ba0410cbbabb"
 
+  # main-usage = "api/v1"
+  # on-revision = ""
+
 # Once we update "github.com/go-openapi/runtime" we can remove this override
 [[override]]
   name = "github.com/go-openapi/analysis"
   revision = "d5a75b7d751ca3f11ad5d93cfe97405f2c3f6a47"
 
+  # main-usage = "api/v1"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/spec"
   revision = "3a0434164aa36744c7ba29b822d36894a1e5ec96"
 
+  # main-usage = "api/v1"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/strfmt"
   revision = "0cb3db44c13bad3b3f567b762a66751972a310cc"
 
+  # main-usage = "swagger and pkg/controllers"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/swag"
   revision = "96d7b9ebd181a1735a1c9ac87914f2b32fbf56c9"
 
+  # main-usage = "api/v1"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/go-openapi/validate"
   revision = "b11dbf889b64cf815a7e2667412df313f4a3f17b"
 
+  # main-usage = "api/v1"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/gorilla/mux"
   revision = "26a6070f849969ba72b72256e9f14cf519751690"
 
+  # main-usage = "plugins/cilium-docker"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/hashicorp/consul"
   revision = "05bc9149a5204b213a61bc410b8cec21f10a23fe"
 
+  # main-usage = "pkg/kvstore"
+  # on-revision = ""
+
 [[constraint]]
   name = "github.com/hashicorp/go-immutable-radix"
-  version = "1.0.0"
+  revision = "v1.0.0"
```

Comment:
Was this a mistake? If so, how did it work before?

Reply:
See [1]